### PR TITLE
[SECSD-469] Refresh MSR material

### PR DESCRIPTION
**Pull request by krysztalzg**
In this PR I've refreshed content on pages regarding _Introduction_ with new flow chart and steps estimation table, _Risk Analysis_, writing the _Report_, _Dictionary_ and _References_.

### CODE REVIEW

File: Pages/01.Introduction.md [hunk 1]
```diff
@@ -50,41 +50,72 @@ This section contains a step-by-step description of the process, as well as an o
...
+1. Code review and static analysis:
+    * Overall review of the project’s code quality, eg. architecture, configuration, tests.
+    * Static analysis with dedicated tools.
+1. Analysis of security requirements:
+    * Manually checking implementation of security measurements for sensitive assets.
+    * Making use of various security breaches and testing tools to check if sensitive data can be extracted from project running on a user's device.
+1. Report creation:
+    * Filling [Review Template](https://docs.google.com/document/d/1haQzOdO_XNQlnKExSxf_jMa_GgUKZdPuwMvP-50Urd0) with collected information about project and performed tests.
+    * Discussing all findings with the project team.
+    * Presenting results of the review on your mobile team meeting to increae security awareness.
```
Comment (mtbrzeski) [inline Pages/01.Introduction.md:73]:
```suggestion
    * Presenting results of the review on your mobile team meeting to increase security awareness.
```

File: Pages/01.Introduction.md [hunk 2]
```diff
@@ -25,9 +25,9 @@ A full analysis of the app can be performed only with all the required elements.
...
+The application (e.g. APK file) and a description are required to analyse the app behaviour and find what kind of data is processed in the application. Description of all the integrations is required to define communication methods and potential sources of data leaks.
 
-The last required element is the application’s source code, this allows us to find potential security bugs in the project and create a list of security elements that should be implemented in the app. List of implemented elements allows to verify if the list met the expected requirements that should be implemented in the project. Mentioned elements should be treated as the inputs for the Mobile security process
+The last required element is the application’s source code, this allows us to find potential security bugs in the project and create a list of security elements that should be implemented in the app. List of implemented elements allows verifiaction if the list met the expected requirements that should be implemented in the project. Mentioned elements should be treated as the inputs for the Mobile security process
```
Comment (mtbrzeski) [inline Pages/01.Introduction.md:30]:
```suggestion
The last required element is the application’s source code, this allows us to find potential security bugs in the project and create a list of security elements that should be implemented in the app. List of implemented elements allows verification if the list met the expected requirements that should be implemented in the project. Mentioned elements should be treated as the inputs for the Mobile security process
```

File: Pages/01.Introduction.md [hunk 3]
```diff
@@ -50,41 +50,72 @@ This section contains a step-by-step description of the process, as well as an o
...
 
 The results of the process should be presented as report containing the full results of the analysis with proper description. The following sections should be provided:
 
-* Security classification containing a set of requirements that the project should obey depending on security level assigned earlier
-* Verification of security requirements implementation, also containing a list of security breaches and missing security elements
-* Quick wins and general suggestions regarding the security improvements that should be implemented to comply with certain security levels requirements
+* Assessment of possible risks for application owner and users.
+* Security classification containing a set of requirements that the project should obey depending on security level assigned.
+* Verification of security requirements implementation, also containing a list of security breaches and missing security elements.
+* Recommendations regarding the security improvements that should be implemented to comply with security level requirements.
```
Comment (mtbrzeski) [inline Pages/01.Introduction.md:121]:
```suggestion
* Recommendations regarding the security improvements that should be implemented to comply with security level requirements. Low hanging fruits can be highlighted in this section, if any 
```

File: Pages/02.Risk_Analysis.md [hunk 4]
```diff
@@ -88,36 +90,41 @@ Keywords related to **Android** system permissions are listed below:
...
 * HomeKit
+* Keychain Sharing
 * Location Services
 * Microphone
 * Motion activity and fitness
 * Music/video activity and the media library
 * Photos
 * Reminders
 * Speech recognition
+* Wallet
```
Comment (mtbrzeski) [inline Pages/02.Risk_Analysis.md:112]: 👏

**Comment by 3Qax** [comment]
Amazing work 👏

**Pull request merged**